Turn the new type scale on by default (#6421)
alphagov/govuk-frontend: CHANGELOG.md

This change was introduced in [pull request #6417: Remove 14 from the type scale](https://github.com/alphagov/govuk-frontend/pull/6417)

> #### The new type scale is on by default

Reviewer: I agree - I reckon the heading should focus on the old one having been removed

We updated our responsive type scale behind the feature flag `$govuk-new-typography-scale` in [GOV.UK Frontend 5.2.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.2.0).

> These type scale changes are now on by default and the `$govuk-new-typography-scale` feature flag has been removed and does not change the behaviour of the type scale.

Reviewer: ```suggestion
These type scale changes are now on by default, and we've removed the `$govuk-new-typography-scale` feature flag.
```
